clojure-ts-mode.el

```diff
@@ -1412,36 +1412,37 @@ if NODE has metadata and its parent has type NODE-TYPE."
 (defun clojure-ts--semantic-indent-rules ()
   "Return a list of indentation rules for `treesit-simple-indent-rules'."
   `((clojure
-     ((parent-is "source") parent-bol 0)
+     ((parent-is "^source$") parent-bol 0)
```

Comment:
Why are the anchors needed?

Reply:
Those matcher helpers actually accept regexps, so for example without `^$` `map_lit` would match both `map_lit` and `ns_map_lit`. I've spent some time debugging why my new rule is not working. There might be other potential types (like `splicing_read_cond_lit` and `read_cond_lit` etc.), so I added anchors explicitly for safety.

Reply:
Might be good to add a few comments about the need for them here and there, so that's clear to people who are not aware of this.

Reply:
Done.